### DISCUSSION TITLE
Click event / destination

### DISCUSSION THREAD

**Discussion by jmmpf** [TarekRaafat/autoComplete.js#194] [Q&A] [answered]
Sorry if I'm missing something super obvious, but if I wanted the selected result to open a page is that built into the script already? I didn't see a way to add URLs to the JSON and didn't see the code in onSelection to open a URL.

Maybe it doesn't and it was just expected for someone using this to be able to add that?
Any help would be appreciated. Would love to just add the URL to JSON file and then have that be what's opened in the current window when a autocomplete selection is clicked.

Thank you ahead of time.

**Comment by TarekRaafat** [comment, marked as answer]
Hello @jmmpf,

Apologies for my late reply.

Assuming you have a `JSON` object that looks like the below.

```json
{
    "key": "value",
    "url": "https://www.google.com/"
}
```

> Option 1: Redirect to URL
```js
onSelection: feedback => {
    window.location.href = feedback.selection.value.url;
}
```

> Option 2: Open new page
```js
onSelection: feedback => {
    window.open(feedback.selection.value.url, "_blank");
}
```

Have a nice day! :)
